package.json

```diff
@@ -43,7 +43,7 @@
     "async-mqtt": "^2.5.0",
```

Comment:
might want to bump async-mqtt to 2.6.0
asnc-mqtt 2.5.0 is still using mqtt 2.3.1: https://github.com/mqttjs/async-mqtt/blob/v2.5.0/package.json#L28